### PR TITLE
Clean-up mangement, router and runtime code.

### PR DESCRIPTION
**Pull request by croudet**

### CODE REVIEW

File: router/src/main/java/io/micronaut/web/router/AnnotatedFilterRouteBuilder.java [hunk 1]
```diff
@@ -41,27 +41,23 @@
...
      *
-     * @param beanContext The bean context
      * @param executionHandleLocator The execution handler locator
      * @param uriNamingStrategy The URI naming strategy
      * @param conversionService The conversion service
      * @param contextPathProvider The server context path provider
      */
     @Inject
     public AnnotatedFilterRouteBuilder(
-            BeanContext beanContext,
```
Comment (jameskleeh) [inline router/src/main/java/io/micronaut/web/router/AnnotatedFilterRouteBuilder.java:58]: This is a breaking change and thus must be reverted

File: runtime/src/main/java/io/micronaut/health/HeartbeatTask.java [hunk 2]
```diff
@@ -37,17 +37,14 @@
...
-    private final HeartbeatConfiguration configuration;
     private final CurrentHealthStatus currentHealthStatus;
 
     /**
      * @param eventPublisher      To publish the events related to heartbeat
-     * @param configuration       The configurations for heartbeat
      * @param currentHealthStatus The current status of health indicator
      */
-    public HeartbeatTask(ApplicationEventPublisher eventPublisher, HeartbeatConfiguration configuration, CurrentHealthStatus currentHealthStatus) {
+    public HeartbeatTask(ApplicationEventPublisher eventPublisher, CurrentHealthStatus currentHealthStatus) {
```
Comment (jameskleeh) [inline runtime/src/main/java/io/micronaut/health/HeartbeatTask.java:46]: This is a breaking change and must be reverted

File: router/src/main/java/io/micronaut/web/router/AnnotatedFilterRouteBuilder.java (hunk 1 above)
Comment (croudet) [reply router/src/main/java/io/micronaut/web/router/AnnotatedFilterRouteBuilder.java]: Argument restored.

File: runtime/src/main/java/io/micronaut/health/HeartbeatTask.java (hunk 2 above)
Comment (croudet) [reply runtime/src/main/java/io/micronaut/health/HeartbeatTask.java]: Argument restored.

**Pull request merged**